README.md

```diff
@@ -143,3 +143,15 @@ engine.push(function(req, res, next, end){
   next() /* This will cause end(res.error) to be called. */
 })
 ```
+
+## Running tests
+
+Build the project if not already built:
+
+```bash
+yarn && yarn build
```

Comment:
Nit: I was a bit surprised to see `yarn &&` in this command - isn't that kinda implied for running any command? `yarn test` won't work as documented in the other repos without installing deps first either.
 
```suggestion
yarn build
```

Reply:
valid. fixed.